kofta/src/app/modules/room-chat/EmoteData.ts

```diff
@@ -1,6 +1,228 @@
 import { EmojiData } from "emoji-mart";
 
 export const customEmojis = [
+  {
+    name: "KomodoHype",
+    short_names: ["KomodoPog"],
```

Comment:
@asxlvm these are the shortnames. so in this case you can just change `KomodoPog` to `komodopog`

Reply:
Okay I see